met-web/src/components/engagement/banner/BannerWithoutImage.tsx

```diff
@@ -54,8 +55,8 @@ const BannerWithoutImage = ({ savedEngagement }: BannerProps) => {
                         m={{ lg: '3em 5em 0 3em', md: '3em', sm: '1em' }}
                         rowSpacing={2}
                     >
-                        <Grid item xs={12}>
-                            <Typography variant="h3">{name}</Typography>
+                        <Grid item xs={12} sx={{ maxHeight: '20em', overflowY: 'auto', overflowX: 'auto' }}>
+                            <Typography variant={isSmallScreen ? 'h3' : 'h4'}>{name}</Typography>
                             <Editor editorState={getEditorState(rich_description)} readOnly={true} toolbarHidden />
```

Comment:
I think this bug is related to the description extending out of the bounds, likely implemented in the banner with image

Reply:
yea that's actually what we did in the banner with the image so I just copied the solution

Reply:
Cool, sorry I didn’t see the scroll attributes

Reply:
All good :)